packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -135,7 +135,7 @@ class _NestedScrollViewState extends State<NestedScrollView> {
     return new CustomScrollView(
       scrollDirection: widget.scrollDirection,
       reverse: widget.reverse,
-      physics: new ClampingScrollPhysics(parent: widget.physics),
+      physics: widget.physics ?? new ClampingScrollPhysics(),
```

Comment:
Yeah we could just require that the physics extend ClampingScrollPhysics for this subclass.

@abarth may have opinions, the crazy physics inheritance magic is his baby. ;-)